integration_test/schemas/v2/notification_schemas.py

```diff
@@ -0,0 +1,220 @@
+from integration_test.schemas.v2.definitions import uuid, personalisation
+
+
+template = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "template schema",
+    "type": "object",
+    "title": "notification content",
+    "properties": {
+        "id": uuid,
+        "version": {"type": "integer"},
+        "uri": {"type": "string"}
+    },
+    "required": ["id", "version", "uri"]
+}
+
+get_notification_response = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "GET notification response schema",
+    "type": "object",
+    "title": "response v2/notification",
+    "oneOf": [
+        {"properties": {
+            "email_address": {"type": "string", "format": "email_address"},
+            "type": {"enum": ["email"]},
+
+            "phone_number": {"type": "null"},
+            "line_1": {"type": "null"},
+            "postcode": {"type": "null"}
+        }},
+        {"properties": {
+            "phone_number": {"type": "string", "format": "phone_number"},
+            "type": {"enum": ["sms"]},
+
+            "email_address": {"type": "null"},
+            "line_1": {"type": "null"},
+            "postcode": {"type": "null"}
+        }},
+        {"properties": {
+            "line_1": {"type": "string", "minLength": 1},
+            "postcode": {"type": "string", "minLength": 1},
+            "type": {"enum": ["letter"]},
+
+            "email_address": {"type": "null"},
+            "phone_number": {"type": "null"}
+        }}
+    ],
```

Comment:
`oneOf` does not work with the required section. 
We made a choice that the response always looks the same, meaning we have both phone_number and email_address in the schema, but only one of them should be